include/mqt-core/operations/CompoundOperation.hpp

```diff
@@ -62,7 +62,10 @@ class CompoundOperation final : public Operation {
 
   std::vector<std::unique_ptr<Operation>>& getOps() { return ops; }
 
-  [[nodiscard]] std::set<Qubit> getUsedQubits() const override;
+  [[nodiscard]] std::set<Qubit>
+  getUsedQubits(const std::function<Qubit(Qubit)>& perm = [](Qubit q) {
```

Comment:
```suggestion
  getUsedQubits(const std::function<Qubit(const Qubit)>& perm = [](const Qubit q) {
```
Just being a bit pedantic here, but I guess the extra `const` doesn't hurt here.
Holds for the other overloads as well.